webextensions/api/menus.json

```diff
@@ -7,7 +7,7 @@
           "support": {
             "chrome": {
               "alternative_name": "contextMenus",
-              "version_added": true
+              "version_added": "7"
```

Comment:
:x: Based on https://github.com/chromium/chromium/commit/34b3680e16699f573c9ecedb580837d1638b8259#diff-a74a7f30d2b4299b1f6e45392b1db1f2a825789af7626a64ebd997ed98b8faaaR49, this landed in [6.0.469.0](https://github.com/chromium/chromium/releases/tag/6.0.469.0), before Chrome [6[.0.472]](https://duckduckgo.com/?q=chrome+6.0.472&t=ffab&ia=web) was released, so it should be in Chrome 6.